pkgs/servers/db-rest/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, nodejs_18
+, nix-update-script
+}:
+let
+  buildNpmPackage' = buildNpmPackage.override { nodejs = nodejs_18; };
+in buildNpmPackage' rec {
+  pname = "db-rest";
+  version = "6.0.2";
+
+  src = fetchFromGitHub {
+    owner = "derhuerst";
+    repo = pname;
+    rev = version;
+    hash = "sha256-xgWMLGk4ksexOg0+KIPEKaW2d16QWdCElAivj/nle0k=";
+  };
+
+  npmDepsHash = "sha256-ZWD1FmGJX+GUSa1h/dDnfXVIWKR6Z92nN5YP3bbGwcU=";
+  dontNpmBuild = true;
+
+  buildPhase = ''
+    runHook preBuild
+    REDIS_URL="" ${nodejs_18}/bin/node ./build/index.js
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/lib
+    cp -r * $out/lib
```

Comment:
Just curious, did the default `npm pack`-based install hook miss something here?

You may also want to `npm prune --omit=dev` manually if you'll be overriding the install hook

Reply:
I'm not sure what I did back then, the default thing should probably be fine